src/main/resources/com/openshift/jenkins/plugins/OpenShiftDSL.groovy

```diff
@@ -900,6 +910,17 @@ class OpenShiftDSL implements Serializable {
             }
         }
 
+        public boolean exists() throws AbortException {
+            return names().size() > 0
+        }
+
+        public void notExists( Closure<?> body ) {
```

Comment:
Did you mean to keep this method around?  I don't think the implementation will work.